packages/dcos-history/extra/history/server_util.py

```diff
@@ -0,0 +1,116 @@
+import logging
```

Comment:
nit: Moving this code to a different file would have been nice to have separate from the code structure changes.